plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -67,12 +71,15 @@ public void execute(IProgressMonitor monitor) throws InvocationTargetException,
       sourcePaths.add(new Path("src/main/java"));
       sourcePaths.add(new Path("src/test/java"));
       javaConfig.setSourceFolders(sourcePaths);
-      
       facetedProject.installProjectFacet(JavaFacet.VERSION_1_7, javaConfig, monitor);
-      facetedProject.installProjectFacet(WebFacetUtils.WEB_25, null, monitor);
-
+      
       CodeTemplates.materialize(newProject, config, progress.newChild(40));
       
+      IDataModel webModel = DataModelFactory.createDataModel(new WebFacetInstallDataModelProvider());
+      webModel.setBooleanProperty(IJ2EEModuleFacetInstallDataModelProperties.ADD_TO_EAR, false);
+      webModel.setBooleanProperty(IJ2EEFacetInstallDataModelProperties.GENERATE_DD, false);
+      webModel.setBooleanProperty(IWebFacetInstallDataModelProperties.INSTALL_WEB_LIBRARY, false);
+      facetedProject.installProjectFacet(WebFacetUtils.WEB_25, webModel, monitor);
```

Comment:
So if we were to make this more Eclipse-WTP'y rather than Maven'y, then we could keep these and instead forcibly overwrite them when we materialize the templates (since we know we're a new project).